src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+const errorHandler = ((err, req, res, next) => {
+    console.log('Error', err);
+    res.send({
+        error: 'not found',
+        message: 'error',
```

Comment:
add message also if you don't get any error from the error stack also use destructuring

Reply:
Done